### PR TITLE
Add GitHub action to update JSDoc

### PR DESCRIPTION
**Pull request by plata**
As explained in #1233, this should not be the final solution. However, it will work for the meantime while not too many PRs must be handled in parallel.

### CODE REVIEW

**Comment by madoar** [comment]
Shouldn't the `docs` folder be excluded?

File: jsdoc_conf.json [hunk 1]
```diff
@@ -6,6 +6,9 @@
     "plugins"      : [],
     "recurseDepth" : 10,
     "source"       : {
+        "exclude" : [
+            "docs"
```
Comment (plata) [inline jsdoc_conf.json:10]: It is excluded.

Comment (madoar) [reply jsdoc_conf.json:10]: But why does the PR contain so many changes to files located in that directory?

Comment (plata) [reply jsdoc_conf.json:10]: Because the jsdoc got updated. It's pushed to that directory and then shown in GitHub pages.

Comment (madoar) [reply jsdoc_conf.json:10]: Oh I see it is the output of the job... Somehow I mixed that up

**Review by madoar — APPROVED** [review]

**Pull request merged**